modules/ROOT/pages/editor-command-identifiers.adoc

```diff
@@ -79,9 +81,12 @@ The commands on the following table are provided by the {productname} editor and
 |Indent |Indents the current selection.
 |Outdent |Outdents the current selection.
 |InsertHorizontalRule |Inserts a horizontal rule at the cursor location or inplace of the current selection.
-|InsertLineBreak |Adds a line break `+<br/>+` at the current cursor or selection.
+|InsertLineBreak |Adds a line break `+<br>+` at the current cursor or selection.
+|InsertParagraph |Inserts a new line at the current cursor or selection.
```

Comment:
We should look into this api in future, perhaps InsertNewLine ?

Reply:
It needs to use the same name as the native command and can't be changed, as that's why this was implemented (since commands no longer fallback to the native variants).